### PR TITLE
use swaybg to set wallpaper and start xdg-desktop-portal-wlr in river init

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: files/system/river/etc/river/init [hunk 1]
```diff
@@ -177,6 +177,9 @@ riverctl spawn "/usr/libexec/xfce-polkit"
 riverctl spawn waybar
 riverctl spawn kanshi
 riverctl spawn nm-applet
+riverctl spawn "swaybg --image /usr/share/backgrounds/f41/default/f41-01-night.png"
+riverctl spawn "/usr/libexec/xdg-desktop-portal-wlr"
+riverctl spawn "/usr/libexec/xdg-desktop-portal"
```
Comment (rehashedsalt) [inline files/system/river/etc/river/init]: Is this required? `xdg-desktop-portal.service` is a user service that should take care of this.

On hyprland I also have `xdg-desktop-portal-hyprland.service`. Is there another for `-wlr`? Are they not being launched when you log in?

**Review by rehashedsalt — COMMENTED** [review]
If the River branch uses Sway tooling and doesn't yet have `swaybg`, this just makes sense.

Not sure about the portal stuff though -- systemd should already be handling that. See comment.

File: files/system/river/etc/river/init (hunk 1 above)
Comment (rehashedsalt) [reply files/system/river/etc/river/init]: These units are required by `graphical-session.target`, so if you're logging into a tty and then spawning river manually, they may not be launched. If they're still not started when you log in through the DM then we have a larger problem.

Comment (ghost) [reply files/system/river/etc/river/init]: hmm i can spawn "systemctl start ...."
~~EDIT: i have not tested the current changes(especially swaybg) yet~~ EDIT5: it works 
~~EDIT2: the default init starts pipewire manually, this should be left to systemd aswell~~ EDIT3: pipewire is automatically started on login systemctl restart could be used for tty users maybe
~~EDIT4: on my river config if i dont start the desktop portal myself it doesnt work~~
~~EDIT5: it needs to be added to the session.slice first before considering tty users~~
~~EDIT6: when i systemctl start --user xdg-desktop-portal-wlr and then systemctl restart --user xdg-desktop-portal it works~~
only thing to make it work is deleting the manual startup of pipewire pipewire-pulse and wireplumber, time to update my commits

Comment (ghost) [reply files/system/river/etc/river/init]: commits have been updated, time for a test

**Comment by ghost** [comment]
im sorry for messing up the commits i will close this PR for the time being

**Pull request closed without merging**